### ISSUE TITLE
Stuck on installing Youtube integration

### ISSUE THREAD

**Issue by Xa4-J** [home-assistant/home-assistant.github.io#28689]
### Feedback

When installing the Youtube integration I  can't select anything at the end.

### URL

https://www.home-assistant.io/integrations/youtube/

### Version

2023.8.4

### Additional information

![Youtube error](https://github.com/home-assistant/home-assistant.io/assets/69121502/49bf5c2b-2be8-4a66-8056-08d8effbfd30)

**Comment by joostlek** [comment]
Is the account you used to login subscribed to channels?

**Comment by Xa4-J** [comment]
I have followed the instructions https://www.home-assistant.io/integrations/youtube
![Youtube API](https://github.com/home-assistant/home-assistant.io/assets/69121502/1f21ad23-2305-458b-8f47-bc500ae3f398)

**Comment by joostlek** [comment]
Yes but I mean, are you subscribed to YouTube channels? Or is this a fresh account?

**Comment by Xa4-J** [comment]
What do you mean with sunscribed to Youtube channels? I use my google account for You tube music (free account) and want to play the URL via Home Assistant.

**Comment by joostlek** [comment]
Ah right. Okay. 2 things.

1. The integration currently just shows you the latest upload and the subscribercount from channels you're subscribed to. (And with subscription I mean the video part of the website)
![image](https://github.com/home-assistant/home-assistant.io/assets/7083755/19b603e7-35dd-44bd-a0fe-bb644d8c542e)
2. You can already play youtube videos using [Cast](https://www.home-assistant.io/integrations/cast/) (if you have a cast device) or maybe using [Media extractor](https://www.home-assistant.io/integrations/media_extractor/) (Although this one seems to have struggles with youtube, I still need to look into that).

So long story short, the youtube integration isn't made for playing youtube videos, but I think you did help me find a bug where I will create a fix for. thanks!

**Comment by Xa4-J** [comment]
Okay, clear. Thank you.
